onnx/onnx_cpp2py_export/checker.pyi

```diff
@@ -17,5 +17,5 @@ def check_attribute(bytes: bytes, checker_context: CheckerContext, lexical_scope
 def check_node(bytes: bytes, checker_context: CheckerContext, lexical_scope_context: LexicalScopeContext) -> None: ...  # noqa: A002
 def check_function(bytes: bytes, checker_context: CheckerContext, lexical_scope_context: LexicalScopeContext) -> None: ...  # noqa: A002
 def check_graph(bytes: bytes, checker_context: CheckerContext, lexical_scope_context: LexicalScopeContext) -> None: ...  # noqa: A002
-def check_model(bytes: bytes, full_check: bool, skip_opset_compatibility_check: bool) -> None: ...  # noqa: A002
-def check_model_path(path: str, full_check: bool, skip_opset_compatibility_check: bool) -> None: ...
+def check_model(bytes: bytes, full_check: bool, skip_opset_compatibility_check: bool, check_custom_domain: bool) -> None: ...  # noqa: A002
```

Comment:
@gramalingam @xadupre @liqunfu do you have an opinion on this new parameter `check_custom_domain` ?

Reply:
Looks like this flag requires (i.e., checks) a schema to be registered for even custom domain ops? If so, it sounds reasonable to me. It would help to add documentation to the code (the C++ code earlier) clarifying the role of the option.

Reply:
I think we need it to preserve the current behaviour otherwise, many tests will fail in many places if new domains are checked by default.